### ISSUE TITLE
Incompatible with OnionOO Document Version 3

### ISSUE THREAD

**Issue by garrettr** [duk3luk3/onion-py#5]
Installed with `pip install onionpy`, got onionpy-0.3.1. Tried the usage example from the README, but the query failed with the following traceback:

```
▶ python
Python 3.5.1 (default, Dec  7 2015, 21:59:08) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.1.76)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from onion_py.manager import Manager
>>> from onion_py.caching import OnionSimpleCache
>>> manager = Manager(OnionSimpleCache())
>>> s = manager.query('summary', limit=4)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/garrett/code/freedomofpress/tor_relay_db/env/lib/python3.5/site-packages/onion_py/manager.py", line 167, in query
    raise MajorVersionMismatchError("Received OnionOO Document with version {}, this library only supports up to version {}".format(versions[0],self.OOO_VERSION_MAJOR))
onion_py.manager.MajorVersionMismatchError: Received OnionOO Document with version 3, this library only supports up to version 2
>>> 
```

**Comment by duk3luk3** [comment]
So the version check is working as intended ;-)

I'll try to upgrade onion-py asap.

**Comment by duk3luk3** [comment]
It's fixed with OnionPy version 0.3.2.

**Comment by duk3luk3** [comment]
@garrettr you still here?
